network/src/peer/mod.rs

```diff
@@ -190,6 +198,13 @@ where
                             }
                         },
                         maybe_message = reader.next() => {
+                            trace!(
+                                NetworkSchema::new(&self.network_context).connection_metadata(&self.connection_metadata),
+                                "{} maybe_message: {:?}",
+                                self.network_context,
+                                maybe_message,
+                            );
```

Comment:
Might be helpful to say, `{} Peer received maybe_message: {:?}`

Reply:
also: since we already have an `info!` if the message is empty, you could move this log bellow in the match case `Some(Ok(message)) =>`

Reply:
it looks like `handle_inbound_message` already does this trace log effectively, I'll just remove this